py_ocsf_models/objects/service.py

```diff
@@ -16,7 +16,7 @@ class Service(BaseModel):
     - Version (version) [Optional]: The version of the service, helping to track changes or updates over time.
     """
 
-    labels: Optional[list[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/service?extensions=. `labels` must be a String Array.